docs/source/index.md

```diff
@@ -1,4 +1,25 @@
-# The Jupyter Notebook
+# Jupyter Notebook Documentation
+
+Welcome to the **Jupyter Notebook** documentation site. **Jupyter Notebook**
+is a lightweight notebook authoring application, and is a part of [Project
```

Comment:
Would "web-based notebook environment for interactive computing" or " interactive computing environment that enables users to author notebook documents" or " web application for creating and sharing computational documents" work better than "lightweight notebook authoring application"? 

References:
1. https://github.com/jupyter/notebook#readme
> The Jupyter notebook is a web-based notebook environment for interactive computing.
2. https://jupyter-notebook.readthedocs.io/en/stable/examples/Notebook/What%20is%20the%20Jupyter%20Notebook.html
>  interactive computing environment that enables users to author notebook documents that include: - Live code - Interactive widgets - Plots - Narrative text - Equations - Images - Video
3. https://jupyter.org/
>  web application for creating and sharing computational documents

Reply:
The guiding principles of these ecosystem PRs are:

- Accessibility to non-software-engineer and non-jupyter-developer audiences (technical professionals like astronomers, chemists, geologists, geneticists...and students, etc)
- Use plain language
- Minimize jargon
- Explain what the product/subproject IS, and HOW it fits under the Jupyter umbrella project

I think some of the alternatives here hurt the text's plain-language properties by adding jargon and increasing word count/verbosity/sentence complexity (environment is meaningful to programmers but probably not to others...the point about interactive computing is already made further down in the same paragraph in a way that more gradually communicates additional info without overloading the text here).

I did get some external feedback, so I'm gonna change "lightweight notebook authoring application" to "simplified notebook authoring application", and we can talk more in the meeting.